corehq/apps/receiverwrapper/util.py

```diff
@@ -154,7 +154,13 @@ def get_commcare_version_from_appversion_text(appversion_text):
         r'"([\d.]+)"\s+\(\d+\)',
         r'"\s*([\d.]+)\s*"',
     ]
-    return _first_group_match(appversion_text, patterns)
+    version = _first_group_match(appversion_text, patterns)
+
+    # Check if the version is in the format of major.minor and append .0 if needed
+    if version and len(version.split('.')) == 2:
```

Comment:
Is there a possibility of no dots at all? Like `version "3"(825679)`? That could be addressed by turning this into a while loop.

nit: Could use `str.count()` here rather than `str.split()` with `len()`.

```suggestion
    while version and version.count('.') < 2:
```

----

If the while loop is unwanted, the suggestion becomes

```suggestion
    if version and version.count('.') == 1:
```

Reply:
Thanks Daniel! I think it is possible that to have only major version. I took your suggestion and added test in 975aaab89c7ecafa6b7d5c5786e86189d2bbccb1